torch/ao/quantization/backend_config/__init__.py

```diff
@@ -17,4 +18,5 @@
     "BackendPatternConfig",
     "DTypeConfig",
     "ObservationType",
+    "get_onednn_backend_config",
```

Comment:
should this change happen in previous PR?

Reply:
OK